### PR TITLE
Adding the troubleshooting section to the README

### PR DESCRIPTION
**Pull request by exceptorr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,6 +128,109 @@ NOTE: You can leave that static server running in one session while you continue
 to execute `tox` on another session. That server will pick up any new changes to
 the report automatically so you don't have to restart it each time.
 
+Troubleshooting
+----------
+
+Since Kubernetes charms are not supported by the `juju debug-hooks`, the only
```
Comment (relaxdiego) [inline README.md:134]: *not supported by `juju debug-hooks`* (remove "the")

File: README.md [hunk 2]
```diff
@@ -128,6 +128,109 @@ NOTE: You can leave that static server running in one session while you continue
...
+way to intercept the control of code execution is to initialize the non-tty-bound
```
Comment (relaxdiego) [inline README.md:135]: *way to intercept code execution* (remove "the control of")

File: README.md [hunk 3]
```diff
@@ -128,6 +128,109 @@ NOTE: You can leave that static server running in one session while you continue
...
+
+For this purpose, we chose the [rpdb](https://pypi.org/project/rpdb/), the
+remote Python debugger based on pdb.
+
+So, given you have already deployed your model and you would like to debug the
+`config-changed` hook, you should do the following:
+
+Given `prometheus` is an application name and `lma` is a Juju model name, 
+execute the following command to grab an interactive Bash shell to the operator
+pod:
```
Comment (relaxdiego) [inline README.md:146]: For example, given that you have already deployed an application named `prometheus` in a Juju model named `lma` and you would like to debug your `config-changed` handler, execute the following:

File: README.md [hunk 4]
```diff
@@ -128,6 +128,109 @@ NOTE: You can leave that static server running in one session while you continue
...
+
+    def on_config_changed_handler(event, fw_adapter):
+        import rpdb
+        rpdb.set_trace()
+        # < ... rest of the code ... >
+
+
+Save the file (`:wq`). Do not close the current shell session!
+
+Open another terminal session to the machine with Juju client and trigger the
```
Comment (relaxdiego) [inline README.md:178]: Can we just say "Open another terminal session in your workstation and trigger the `config-changed` hook as follows:"

File: README.md [hunk 5]
```diff
@@ -128,6 +128,109 @@ NOTE: You can leave that static server running in one session while you continue
...
+    -> self._reemit(event_path)
+      /var/lib/juju/agents/application-prometheus/charm/lib/ops/framework.py(745)_reemit()
+    -> custom_handler(event)
+      /var/lib/juju/agents/unit-prometheus-0/charm/hooks/config-changed(68)on_config_changed()
+    -> on_config_changed_handler(event, self.fw_adapter)
+    > /var/lib/juju/agents/unit-prometheus-0/charm/hooks/config-changed(91)on_config_changed_handler()
+    -> set_juju_pod_spec(fw_adapter)
+    (Pdb)
+
+
```
Comment (relaxdiego) [inline README.md:231]: From this point forward, the usual `pdb` commands apply.  For more information on how to use `pdb`, see the official [pdb documentation](https://docs.python.org/3/library/pdb.html).

**Comment by exceptorr** [comment]
@relaxdiego Thanks for the comments! I've addressed them so it's ready to be reviewed again. :)

**Comment by relaxdiego** [comment]
LGTM thanks!

**Pull request merged**